frontend/portals/ShareButton/index.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import isiOSTheme from '../../helpers/isiOSTheme';
+import ShareButton from '../../components/ShareButton';
+
+export default props => <ShareButton {...props} iOSTheme={isiOSTheme} />;
```

Comment:
I understand this is a way to avoid too much mocking within the unit tests, but I'd personally prefer to just import the helper inside the component.

The result could be passed to a portal though, if we decide to support one.

Reply:
Updated component to use helper within